x-pack/plugins/security_solution/common/endpoint/types/trusted_apps.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { TypeOf } from '@kbn/config-schema';
+import { GetTrustedAppsRequestSchema } from '../schema/trusted_apps';
+import { ExceptionListItemSchema } from '../../../../lists/common';
+import { FoundExceptionListItemSchema } from '../../../../lists/common/schemas/response';
+
+/** API request params for retrieving a list of Trusted Apps */
+export type GetTrustedAppsListRequest = TypeOf<typeof GetTrustedAppsRequestSchema.query>;
+export type GetTrustedListAppsResponse = Pick<
+  FoundExceptionListItemSchema,
+  'per_page' | 'page' | 'total'
+> & {
+  data: TrustedApp[];
+};
+
+/** Type for a new Trusted App Entry */
+export type NewTrustedApp = Pick<
+  ExceptionListItemSchema,
+  'created_at' | 'created_by' | 'name' | 'description' | 'entries'
+> & {
+  os: string;
```

Comment:
Yeah, it will probably be an `enum` - just need to find the one that List is using (if one exists) and reuse it or come up with our own.  FYI: this `os` field does not exist today on the data storage schema - the Trusted Apps API will populate it from the `_tags[]` array.

Reply:
we could introduce an enum with with the Create flow and validation

Reply:
We have a ticket to add the `os` to the exception list item schema in 7.10

Reply:
Yes, we can create the Enum either with the UI changes or when we add the Create api.

@peluja1012 sounds good. Madi had mentioned that to me, so we're aware we will have to adjust our "translator" once that gets introduced.